.travis.yml

```diff
@@ -13,5 +13,3 @@ install:
 # command to run tests
 script:
   - pytest -v backend/test_*.py
-  # - coverage run -m  pytest -v
```

Comment:
what did the 'coverage' do in previous iterations?

Reply:
Nothing. They were in the professor's sample file, but we couldn't get them to work, so we had commented them out.